.github/workflows/linterns.yaml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
```

Comment:
- [ ] You did great setting up linters, But two of your linters are failing, I'll suggest you fix that as all your linters need to have agreen checkmark :heavy_check_mark: Also the name of this file should be `linters.yml` and not `linterns.yml`. :smile: